web/components/Customize/CharacterList/Icon.tsx

```diff
@@ -7,7 +7,10 @@ import NextLink from "next/link";
 import IconImage from "../IconImage";
 
 const Icon = ({ visible = true, character }: { visible?: boolean; character: Character }) => {
-  const [existing] = useConfig("characters");
+  const [charactersWeekly] = useConfig("charactersWeekly");
+  const [characters] = useConfig("characters");
+  const existing = [...new Set([...charactersWeekly, ...characters])];
```

Comment:
This line instantiates a new array every render, making the `useMemo` below pointless. The set should be moved inside the memo and call `has` on it instead of converting it to array in a roundabout way.